sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -0,0 +1,341 @@
+<!-- Automatically generated byExpressionsSchemaSuite -->
+## Summary
```

Comment:
Hey @cloud-fan and @gatorsmile, let's don't add a generated file into a main repo.